packages/ui/src/components/editor/properties/trigger/index.tsx

```diff
@@ -78,9 +86,14 @@ const TriggerProperties: EditorComponentType = (props) => {
     for (const entity of callbackQuery) {
       if (!hasComponent(entity, EntityTreeComponent)) continue
       const callbacks = getComponent(entity, CallbackComponent)
+      const label = getOptionalComponent(entity, NameComponent)
```

Comment:
why not just include these in the query?

Reply:
good point, just did a spot-fix